### ISSUE TITLE
Edit breakpoints (Ctrl+E) doesn't work with multiple selection

### ISSUE THREAD

**Issue by justanotheranonymoususer** [x64dbg/x64dbg#2117]
You can select multiple breakpoints and click on Edit, but only one breakpoint is affected.

Related to: https://github.com/x64dbg/x64dbg/issues/1304

**Comment by torusrxxx** [comment]
Yeah, and because each "edit breakpoints" action invokes many commands to set the attribute of breakpoint, it might become very slow if you select huge number of breakpoints and edit them.

**Comment by 77ZaRR77** [comment]
Very similar bug also appears in Breakpoints tab. SHIFT+mouse selecting all breakpoints then right mouse button will show operations with breakpoints, but selecting them all with SHIFT+A then right mouse button won't show thoose..

**Comment by eltimen** [comment]
> Very similar bug also appears in Breakpoints tab. SHIFT+mouse selecting all breakpoints then right mouse button will show operations with breakpoints, but selecting them all with CTRL+A then right mouse button won't show thoose.. At current last version: Aug 7 2020.
This behavior is observed when the first row in the selection is the "breakpoint type header":
![изображение](https://user-images.githubusercontent.com/53598853/178969539-5e7c3691-51d3-443f-90c9-c01ad77d8842.png)
But the breakpoint actions are shown in the case below:
![изображение](https://user-images.githubusercontent.com/53598853/178969706-4cf703ad-4650-4744-9322-74aa9b8f51aa.png)
